uitools/cpp/Esri/ArcGISRuntime/Toolkit/SearchViewController.h

```diff
@@ -126,12 +130,16 @@ namespace Toolkit {
     void selectedResultChanged();
     void isEligableForRequeryChanged();
     void isAutomaticConfigurationEnabledChanged();
+    void thresholdRatioRepeatSearchChanged();
 
   private:
     void onQueryChanged();
     void onSourcesAdded(const QModelIndex& parent, int firstSource, int lastSource);
     void onSourcesRemoved(const QModelIndex& parent, int firstSource, int lastSource);
 
+    bool checkPanningDifferenceLastSearch(Geometry& geom);
+    bool checkZoomingDifferenceLastSearch(Geometry& geom);
```

Comment:
🔧  Good stuff making these! Couple of points.

* Arguments should be `const Geometry&` not `Geometry&`.
* AFAIK these don't require the scope of `this` so could exist entirely within the anonymous namespace of the C++ file.
* OR if you do need to keep these as class methods, please mark the methods as `const`.

Reply:
The way I structured is that they depend on two internal variables (`threshold` and the `lastsearcharea`), so I prefer to keep them in the class scope. Adding the missing const.